src/background/httpClients.ts

```diff
@@ -130,3 +140,55 @@ export const httpClient = {
     return fetch(url, { credentials: 'omit' });
   },
 };
+
+type NonNullFields<T> = {
+  [P in keyof T]: NonNullable<T[P]>;
+};
+
+const _getTeamsInfoQuery = gql`
+  query getTeamsInfo {
+    locales {
+      code
+      name
+      approvedStrings
+      pretranslatedStrings
+      stringsWithWarnings
+      stringsWithErrors
+      missingStrings
+      unreviewedStrings
+      totalStrings
+    }
+  }
+`;
+
+interface GetTeamsInfoResponse {
+  locales: NonNullFields<Required<NonNullFields<GetTeamsInfoQuery>>['locales']>;
+}
+
+const _getProjectsInfoQuery = gql`
+  query getProjectsInfo {
+    projects {
+      slug
+      name
+    }
+  }
+`;
+
+export interface GetProjectsInfoResponse {
+  projects: NonNullFields<
+    Required<NonNullFields<GetProjectsInfoQuery>>['projects']
+  >;
+}
+
+export function graphqlClient(pontoonBaseUrl: string) {
+  const client = getSdk(
+    new GraphQLClient(pontoonGraphQL(pontoonBaseUrl), {
+      method: 'GET',
+    }),
+  );
+  return {
+    getTeamsInfo: client.getTeamsInfo as () => Promise<GetTeamsInfoResponse>,
+    getProjectsInfo:
+      client.getProjectsInfo as () => Promise<GetProjectsInfoResponse>,
+  };
```

Comment:
This wrapping and type juggling above is required because Pontoon's schema define everything as nullable or even optional.

Reply:
This https://docs.graphene-python.org/en/latest/types/list-and-nonnull/ would probably solve it, but I don't know if the GQL cannot actually returns nulls in the lists etc.